src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Unix.cs

```diff
@@ -120,10 +120,10 @@ private static int GetEbcdicCodePage(string cultureName)
             return ebcdicCodePage == -1 ? CultureData.Invariant.EBCDICCodePage : ebcdicCodePage;
         }
 
-        internal bool IsWin32Installed => false;
+        internal static bool IsWin32Installed => false;
 
         internal static unsafe CultureData GetCurrentRegionData() => CultureInfo.CurrentCulture._cultureData;
 
-        private bool ShouldUseUserOverrideNlsData => false;
+        private static bool ShouldUseUserOverrideNlsData => false;
```

Comment:
It's strange for this one to be static on Unix but instance on Windows.

Reply:
Why? It's private property so I see that as an implementation detail. Would you prefer to if-def out the code instead?

Reply:
I'm not saying we can't do it, just that it's strange. It's part of an abstraction layer, where the shared layer expects the same API shape to be available regardless of compilation target. The static vs instance distinction changes the access patterns allowed based on the build.